mount_linux.go

```diff
@@ -126,10 +126,11 @@ var mountflagopts = map[string]func(uintptr) uintptr{
 var errFallback = errors.New("sentinel: fallback to fusermount(1)")
 
 func directmount(dir string, cfg *MountConfig) (*os.File, error) {
-	dev, err := os.OpenFile("/dev/fuse", os.O_RDWR, 0644)
+	fd, err := syscall.Open("/dev/fuse", syscall.O_RDWR, 0644)
```

Comment:
I'd also recommend a comment here saying why this isn't an os.OpenFile call

Reply:
Done.